app/web/src/components/Workspace/CommandModal.vue

```diff
@@ -78,6 +80,50 @@ class Pan implements Command {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface GotoView extends Command {}
+class GotoView implements Command {
+  name = "Goto View";
+  shortcut = "G";
+  expects: CommandArg[] = ["view"];
+  constructor() {
+    this.choices = [];
+  }
+  execute() {
+    const viewId = this.choices[0]?.value;
+    if (!viewId) throw new Error("ViewId Expected");
+    viewStore.selectView(viewId);
+  }
+  // I can't make this static because the instance won't have a reference to it
+  // eslint-disable-next-line class-methods-use-this
+  factory() {
+    return new GotoView();
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface NewView extends Command {}
+class NewView implements Command {
+  name = "New View";
+  shortcut = "N";
+  expects: CommandArg[] = ["stringInput"];
```

Comment:
@jkeiser you might find this part interesting—I added support to type (and hit enter) on an arbitrary string input (here: a name for a view) that is not a part of a dropdown menu